model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/LiquibaseJpaUpdaterProvider.java

```diff
@@ -112,7 +119,30 @@ private void update(Connection connection, File file, String defaultSchema) {
         }
     }
 
-    protected void updateChangeSet(Liquibase liquibase, String changelog, Writer exportWriter) throws LiquibaseException, IOException {
+    protected void updateChangeSet(Liquibase liquibase, Connection connection, String defaultSchema, Writer exportWriter) throws LiquibaseException, SQLException {
+        String changelog = liquibase.getChangeLogFile();
+        Database database = liquibase.getDatabase();
+
+        DatabaseMetaData metadata = connection.getMetaData();
+        String changelogTableName = database.getDatabaseChangeLogTableName();
+        ResultSet changelogTable = metadata.getTables(null, defaultSchema, changelogTableName, null);
+
+        // if changelog table exist
+        if (changelogTable.next()) {
+            ResultSet rs = metadata.getColumns(null, defaultSchema, changelogTableName, null);
+
+            // create DEPLOYMENT_ID column if it doesn't exist
+            if (!hasColumn(rs, DEPLOYMENT_ID_COLUMN)) {
+                logger.infov("Adding missing column {0} to {1} table", DEPLOYMENT_ID_COLUMN, changelogTableName);
+
+                SqlStatement sqlStatement = new AddColumnStatement(null, defaultSchema, changelogTableName, DEPLOYMENT_ID_COLUMN, "varchar(10)", null, null);
```

Comment:
The column should be nullable, see https://github.com/liquibase/liquibase/commit/b16a5d97086f9ac1c0efdf983fdb1667376699b3#diff-4aa547987494c9c61353be5bd6406a5bR180